Script/ResourceFinder.gd

```diff
@@ -1,5 +1,13 @@
 class_name ResourceFinder
 
+# TODO: Fetch this list of recognised image format extensions from the engine.
+# You might might hope that you could use
+# ResourceLoader.get_recognized_extensions_for_type("Image") or
+# ResourceLoader.get_recognized_extensions_for_type("Texture2D"),
+# but you would be disappointed.
+const _IMAGE_EXTENSIONS = [".png", ".jpg", ".jpeg", ".png", ".svg", ".webp"]
```

Comment:
This says .png twice ;) It's a silly thing and I wanted to merge this so I just went and fixed it.